mvp-spec/audit-log-design.md

```diff
@@ -38,26 +15,12 @@ of the Addressable Content.
 4. Shuffle the list of Transmission Results
 5. Build the Audit Log by following the specification below.
 
-The Audit Log follows this structure:
-<!--partial-begin { "files": [ "audit-log-table.md" ] } -->
-<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
-| Field         | Type                         | Detail                        |
-|---------------|------------------------------|-------------------------------|
-| data          | Prebid SSO Data Object       | List the Pseudonymous-Identifiers and the Preferences of the user. |
-| seed          | Seed Object                  | The Seed object is the association of an Addressable Content to the Prebid SSO Data. |
-| transmissions | List of Transmission Results | A list of Transmission Results |
-<!--partial-end-->
-
-### The Transmission Result object
-
-A Transmission Result is the output of a Transmission. It can be a
-Transmission Response without the "children" field when the Receiver of 
-a Transmission responded correctly to a Transmission Request. 
-Those Transmission Results are appended to the Audit Log. Therefore,
-the Contracting Party handles those objects.
-
-### Example of Audit Log
+| Type  | Format|
+|----------|-------|
+| Audit Log  | [audit-log.md](./model/audit-log.md)  |
 
+<details>
+<summary>Audit Log Example</summary>
 <!--partial-begin { "files": [ "audit-log.json" ], "block": "json" } -->
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 ```json
```

Comment:
For some reason, GitHub doesn't render this correctly.
You might need to add extra empty lines around the code block.